### ISSUE TITLE
different gaussian laser profile shape in direct and antenna method

### ISSUE THREAD

**Issue by Gholipoorr** [fbpic/fbpic#676]
Hello in the following script gaussian laser shape is different for the direct and antenna methods, is there a problem here?
[lwfa.txt](https://github.com/fbpic/fbpic/files/13841484/lwfa.txt)
![g_antenna](https://github.com/fbpic/fbpic/assets/77463402/dc108d49-4179-4e13-a72f-76ffc8c0c19d)
![g](https://github.com/fbpic/fbpic/assets/77463402/6ba77a52-e6c2-4f82-96e3-b528342774ad)

**Comment by mishradinkar** [comment]
Did you checked it on the same iteration steps, because in the figures attached above direct one is at 0 iteration

**Comment by RemiLehe** [comment]
I agree with @dinkarm: this should probably be checked at a later timestep.
If this is still an issue, feel free to post your input script, so that we can reproduce the issue.
